pkg/lib/service.js

```diff
@@ -292,7 +292,20 @@
 
         function call_manager_with_reload(method, args) {
             return call_manager(method, args).then(function () {
-                return call_manager("Reload", [ ]);
+                var dfd = cockpit.defer();
+                call_manager("Reload", [ ]).
+                    done(function () { dfd.resolve(); }).
+                    fail(function (error) {
+                        // some systemd versions disconnect too fast from the bus
+                        // (https://bugzilla.redhat.com/show_bug.cgi?id=1560549)
```

Comment:
This line should look like this for easy grepping, tracking:

```
HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1560549
```

Reply:
Ack, changed accordingly.